### PR TITLE
Quiz question animation

### LINKED ISSUE #129: Question popups updates
## Use Case

1. Users are not seeing changing content possibly because questions animate in instantly.
2. During user test comeone remarked that the success message went away too quickly and we agree

## Acceptance Criteria

1. Add a very short delay and then have questions slide in at a slow but reasonably slow rate from the bottom of screen.
2. Extend the success message popup disappear delay by maybe double?
3. Add ability to close manually via "X" in upper right of popup.

### PR DESCRIPTION
**Pull request by meganrm**
Problem
=======
Estimated review size: medium
The questions coming in draw so much attention the users don't notice they've moved forward a page. 
closes #129 

Solution
========
animate the questions in instead of appearing 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. bun dev 
2. click 'Molecular View'
3. go to page 8 
4. question will animate in
5. answer correctly (C) 
6. success message will animate out

### CODE REVIEW

File: src/components/AdminUi.tsx [hunk 1]
```diff
@@ -18,8 +18,8 @@ const AdminUI: React.FC<AdminUIProps> = ({
     totalPages,
 }) => {
     const { page, setPage } = useContext(SimulariumContext);
-    const [visible, setVisible] = React.useState<boolean>(false);
-
+    const isDev = process.env.NODE_ENV === "development";
+    const [visible, setVisible] = React.useState<boolean>(isDev);
```
Comment (meganrm) [inline src/components/AdminUi.tsx:22]: to auto open the admin ui when developing since I use it so much

File: src/components/viewer.module.css [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+            var(--content-height) + (var(--border-thickness) * 2)
     );
 }
 
 .container.example {
-    height: calc(100vh - var(--nav-bar-height) - var(--content-height));
+    height: calc(
+        100vh - var(--nav-bar-height) - var(--content-height) +
+            (var(--border-thickness) * 2)
+    );
```
Comment (meganrm) [inline src/components/viewer.module.css:12]: in doing this, I noticed there was a 2 px space on the bottom that I wanted to remove (caused by not including 2 borders in height calc)

File: src/components/quiz-questions/SuccessFeedback.tsx [hunk 3]
```diff
@@ -4,40 +4,51 @@ import { CheckCircleOutlined, CloseOutlined } from "@ant-design/icons";
...
 import { IconButton } from "../shared/ButtonLibrary";
+import classNames from "classnames";
+import { BG_DARK } from "../../constants/colors";
 
 const SuccessFeedback: React.FC<FeedbackProps> = ({
     title = "That's correct!",
     message,
-    resetForm,
 }) => {
+    const [hasBeenSeen, setHasBeenSeen] = React.useState(false);
```
Comment (meganrm) [inline src/components/quiz-questions/SuccessFeedback.tsx:14]: this replaced the need for the "resetForm"

**Review by ShrimpCryptid — APPROVED** [review]
Left a comment but otherwise LGTM

File: src/components/quiz-questions/popup.module.css [hunk 4]
```diff
@@ -1,18 +1,36 @@
...
     padding: 16px;
+    width: 100%;
     border-radius: 4px 4px 0 0;
     z-index: var(--viewer-overlay-layer);
 }
 
-.container.collapsed {
+.below-view {
+    bottom: -400px;
+}
```
[the remark this answers is not in the document]
Comment (meganrm) [reply src/components/quiz-questions/popup.module.css:21]: no I tried what I could think of and it didnt' work

**Review by toloudis — APPROVED** [review]
LGTM

**Pull request merged**